### PR TITLE
Make `s3 version` work reliably

### PR DESCRIPTION
**Pull request by paulmelnikow**
I read that note in PEP-420, and ignored it. (Nah. It'll be fine.) Now that I've seen this problem manifest I will play along. 😄 

```
Traceback (most recent call last):
  File "/usr/local/bin/s3", line 152, in <module>
    if args.command == 'version': print baiji.__version__
AttributeError: 'module' object has no attribute '__version__'
Traceback (most recent call last):
  File "/usr/local/bin/s3", line 152, in <module>
    if args.command == 'version': print baiji.__version__
AttributeError: 'module' object has no attribute '__version__'

Your `s3` from core is  but 1.2 is required.

Please pull latest core.

rake aborted!
```

### CODE REVIEW

File: baiji/package_version.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This version number is stored here instead of in `__init__.py` because
+# namespace packages and the packages which live inside them can be imported
+# in any order, which means their contents are supposed to be identical. If
+# their contents differ, Python makes no guarantee about which one is loaded.
+#
+# See https://www.python.org/dev/peps/pep-0420/#namespace-packages-today
+
+__version__ = '2.2.8'
```
Comment (paulmelnikow) [inline baiji/package_version.py]: Bump patch.

**Comment by algrs** [comment]
This means we can no longer say `baiji.__version__`, right? Do we do that anywhere but `s3 version`? 

👍

**Comment by paulmelnikow** [comment]
Yes, that's right. That seems regrettable which is why I did not do it before.

We don't reference it anywhere else in code that I'm aware, other than setup.py. We don't log it in the job.

**Comment by algrs** [comment]
Yeah, that's sad, but probably the right thing to do...

**Pull request merged**